lage.config.js

```diff
@@ -0,0 +1,37 @@
+module.exports = {
+  pipeline: {
```

Comment:
Is there documentation on what these settings do?

Reply:
Yes: https://microsoft.github.io/lage/guide/config.html

It's not as comprehensive as I would like, but we do have some docs on this.

Reply:
Oh okay. Maybe put a link in a file comment for ease of finding it?

Also am I understanding right that leading `^` means "do dependencies first"?

Reply:
Oh also can you add docs to the page for `ignore` and `cacheOptions`? (I'm not sure what the `ignore` things are being ignored from)

Reply:
> Oh okay. Maybe put a link in a file comment for ease of finding it?
> 
> Also am I understanding right that leading `^` means "do dependencies first"?

Yes that's it!

> Oh also can you add docs to the page for ignore and cacheOptions? (I'm not sure what the ignore things are being ignored from)

I'll add some comments here in this file first - will add docs later. I'm organizing myself to be able to work on these docs this week.